packages/edit-site/src/components/page-pages/index.js

```diff
@@ -133,7 +133,9 @@ export default function PagePages() {
 						</VStack>
 					);
 				},
-				filters: [ { id: 'search', type: 'search' } ],
+				filters: [
```

Comment:
So this is the global filter and yet we apply it in `title`. This should work in any field right? Maybe it's better to have flag for that and add it in the main component?

Reply:
Yeah, `search` is weird. See related conversation https://github.com/WordPress/gutenberg/pull/55270#discussion_r1362600986 

I'm trying to run with the idea we talked about: `filters` are part of the `fields` API. Though, the more I think about it, the more it makes sense that filters are an independent property of the `DataViews` component:

```js
<DataViews
  fields={fields}
  view={view}
  filters={filters}
  actions={actions}
  ...
/>
```

Reply:
Yeah, I guess will see how the API will evolve. In general a global filter could be part of the API by just having an interface, consumers can abide to. So they would have to map keys(like `search`) with their own respective param.

Reply:
I thought I'd extract `search` from the fields and pass it directly as a prop, as you suggested https://github.com/WordPress/gutenberg/pull/55475